src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/NativeLibrary.NativeAot.cs

```diff
@@ -116,102 +114,13 @@ private static IntPtr LoadFromPath(string libraryName, bool throwOnError)
             return ret;
         }
 
-        private static IntPtr LoadLibraryHelper(string libraryName, int flags, ref LoadLibErrorTracker errorTracker)
```

Comment:
`LoadFromPath` is passing `LOAD_WITH_ALTERED_SEARCH_PATH` to this method in CoreCLR VM copy. Should we do the same?